src/actions/authActions.js

```diff
@@ -109,15 +109,15 @@ export const updateFcmTokenAction = (walletId: string) => {
   return async (dispatch: Dispatch, getState: GetState, api: SDKWrapper) => {
     const fcmToken = await firebaseMessaging.getToken().catch(e => {
       // We was unable to fetch the FCM token.
-      reportLog(`Unable to fetch Firebase FCM token: ${e.message}`, e);
+      reportLog(`Unable to fetch Firebase FCM token: ${e.message}`, e); // eslint-disable-line i18next/no-literal-string, max-len
```

Comment:
It seems the most recent version of eslint-plugin-i18next fixes the ignore options for template literals in `no-literal-string`. Is there a reason not to use it? Also, if it's the _eslint-disable_ comment that causes `max-len` to trigger, I'd try `eslint-disable-next-line`.

Reply:
Updated :) Somehow I've missed this update.
It solves many literal string validation issues.
Yet it seems like it now ignores nested string values in objects that have properties named in all caps.
That's not that common in our project, but it's good to have this in mind. 
I'll put it into translations readme